runner.js

```diff
@@ -37,7 +37,8 @@ function printStatus (status) {
  * Returns the numeric part of the URL.
  */
 function getUrlNumericPart (url) {
-    return parseInt(url.match(/\/(\d+)/)[1]);
+    console.log(url);
+    return parseInt(url.match(/\/(\d+)\/$/)[1]);
```

Comment:
Changing the regex to `/\/(\d+)[\/$]/` should solve the issue without having to change any other lines in the code. Could you do that?